frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -462,12 +490,67 @@ describe('funnelLogic', () => {
                 })
         })
 
-        it('isPropertyExcluded returns true initially, then false when excluded', async () => {
+        it('isPropertyExcluded returns true initially, then false when excluded, and is persisted to team config', async () => {
+            userLogic.mount()
+
+            logic.actions.setPropertyNames(logic.values.allProperties)
```

Comment:
following up from prev comment, I think we shouldn't need to `setPropertyNames` to figure out if a property is excluded or not.